src/components/marker.js

```diff
@@ -53,14 +53,25 @@ export default class Marker extends BaseControl {
     const {className, longitude, latitude, offsetLeft, offsetTop} = this.props;
 
     const [x, y] = this.context.viewport.project([longitude, latitude]);
-    const containerStyle = {
-      position: 'absolute',
+
+    const {left, top} = {
       left: x + offsetLeft,
       top: y + offsetTop
     };
 
+    const isVisible = !(top < 0 || left < 0);
+
+    let containerStyle = {};
+
+    if (isVisible) {
+      containerStyle = {
+        transform: `translate3d(${left}px, ${top}px, 0)`
```

Comment:
Why is CSS transform preferable than absolute positioning? Is it contributing to perf at all?

Reply:
CSS transform is more productive property which uses GPU across rendering. I've complete some test with 50-100-150 markers on Mobile and it shows 2x better performance iOS Safari (Latest) / Android Chrome (Latest)

Reply:
Shall we apply this technique to the Popup component as well?

Reply:
Yes, I think each component should be transformed as well